person.rb

```diff
@@ -1,7 +1,10 @@
-class Person
+require_relative 'nameable'
+
```

Comment:
- [ ] [OPTIONAL] When trying to run the below lines the user gets the error shown in the printscreen
```
person = Person.new(22, 'maximilianus')
p person.correct_name
 capitalizedPerson = Capitalize.new(person)
p capitalizedPerson.correct_name
capitalizedTrimmedPerson = Trimmer.new(capitalizedPerson)
p capitalizedTrimmedPerson.correct_name
```
![afbeelding](https://user-images.githubusercontent.com/39240820/232849097-82d5172f-442c-4c55-9426-cce33bb61cb5.png)

the error occurs because you're not using require_relative for the decorator files, if you would include the below lines of code your application would run as intended.
```
require_relative 'base_decorator'
require_relative 'capitalize_decorator'
require_relative 'trimmer_decorator'
```